agr_literature_service/api/routers/topic_entity_tag_router.py

```diff
@@ -132,6 +132,14 @@ def show_all_reference_tags(curie_or_reference_id: str,
                                                          column_values)
 
 
+@router.get('/by_mod/{mod_abbreviation}',
+            status_code=200)
+def get_reference_tags(mod_abbreviation: str,
```

Comment:
Same as above, I would rename this endpoint to make it clear that this is for all tets for all references for a mod

Reply:
Fixed =)